pkg/sql/sessiondatapb/local_only_session_data.proto

```diff
@@ -226,6 +226,9 @@ message LocalOnlySessionData {
   // and joins using the same default number of bytes per column instead of
   // column sizes from the AvgSize table statistic.
   bool cost_scans_with_default_col_size = 61;
+  // QualityOfService is the session settings interface to admission control,
+  // to control resource overload by SQL queries.
+  sint32 quality_of_service = 62;
```

Comment:
can we make this `[(gogoproto.casttype)="github.com/cockroachdb/cockroach/pkg/util/admission.WorkPriority"]`, which means all these int32s in the code are typed in, e.g. `NewTxnWithSteppingEnabled`